assets/src/domain/shared/entities/tickets/helpers/status.ts

```diff
@@ -0,0 +1,24 @@
+import { Ticket } from '@edtrServices/apollo/types';
+import isTrashed from '../../../services/predicates/isTrashed';
+import { TICKET_STATUS_ID } from '../constants';
+
+const status = (ticket: Ticket) => {
+	if (isExpired(ticketEntity)) {
+		return TICKET_STATUS_ID.EXPIRED;
+	}
+	if (isOnSale(ticketEntity)) {
+		return TICKET_STATUS_ID.ONSALE;
+	}
+	if (isPending(ticketEntity)) {
+		return TICKET_STATUS_ID.PENDING;
+	}
+	if (isSoldOut(ticketEntity)) {
+		return TICKET_STATUS_ID.SOLD_OUT;
+	}
+	if (isTrashed(ticket)) {
```

Comment:
@tn3rb originally this was checking `isArchived`, but since we've decided to go with trashable entities I've used `isTrashed`.
Let me know if this mapping is ok?